src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -71,3 +73,52 @@
     }
   }
 }
+/*******************************************************************************
+* color overrides. Here we recreate the sphinx-design rules for colors
+* (including button and badge colors), and assign our own --pst-color-*
+* variables instead of --sd-color-*.
+*/
+$sd-semantic-color-names: (
+  "primary",
+  "secondary",
+  "success",
+  "info",
+  "warning",
+  "danger",
+  "light",
+  "muted",
+  "dark",
+  "black",
+  "white"
+);
+
+html[data-theme="light"],
+html[data-theme="dark"] {
+  @each $name in $sd-semantic-color-names {
```

Comment:
the SCSS magic is strong with you ✨ 

Can you add a link to the SCSS docs that explain how `@each` works? I think many developers won't be familiar with it so it will be helpful to have breadcrumbs to help them learn quickly if they hit this code

Reply:
> Can you add a link to the SCSS docs that explain how `@each` works?

done